integration-tests/tests/src/test/java/com/squareup/anvil/test/Utils.kt

```diff
@@ -4,3 +4,7 @@ import kotlin.reflect.KClass
 
 internal infix fun KClass<*>.extends(other: KClass<*>): Boolean =
   other.java.isAssignableFrom(this.java)
+
+internal fun Array<KClass<*>>.withoutAnvilModule(): List<KClass<*>> = toList().withoutAnvilModule()
+internal fun Collection<KClass<*>>.withoutAnvilModule(): List<KClass<*>> =
+  filterNot { it.qualifiedName!!.startsWith("anvil.module") }
```

Comment:
MODULE_PACKAGE_PREFIX?

Reply:
Does not work here, because this module doesn't see code from a Kotlin compiler plugin. It's a different classpath.